aerogear/mobile-security-service-operator
# feat(finalizer) : Following the recommendations over how to work with finalizers

## Motivation
https://issues.jboss.org/browse/AEROGEAR-9161
## What
* Remove the finalizer metadata from the YAML file and add it in the reconcile process

## Why
* Following the operator-framework recommendations

## Verification Steps
 
1. Install the operator locally via the command `make create-all` by using this PR and image `docker.io/cmacedo/mobile-security-service-operator:AEROGEAR-9161`
2. Create the namespace for the apps `oc new-project X`
3. Create the app by running `make create-app`
4. Check if the app was created with success ( config map) but it will not be able to save the app in service regards the Oauth impl. 
5. Now, in the same namespace run `make delete-app` it should run but be stuck since it is missing the access to the Service provided by Oauth

## Checklist:


## Progress


## Additional Notes
